app/helpers/template_helper.py

```diff
@@ -78,6 +78,7 @@ def render_template(template, **kwargs):
         language_code=get_locale().language,
         survey_title=cookie_session.get("survey_title"),
         cdn_url=cdn_url,
+        data_layer=get_data_layer(cookie_session.get("theme")),
```

Comment:
It's probably worth defining a theme variable here, given the number of times the cookie_session is accessed

Reply:
Done, thanks!